jenkins/release-workflows/release-notes-generate.jenkinsfile

```diff
@@ -47,7 +47,6 @@ pipeline {
         )
         string(
             name: 'REF',
-            defaultValue: 'main',
```

Comment:
Does it requires to add a branch / ref name now?

Reply:
No, the description says defaults to whatever is in the input manifest. @rishabh6788 can confirm.

Reply:
yeah, defaults to ref in manifest, which should be main before branch cut, right?